workitem/link/link_revision_repository_blackbox_test.go

```diff
@@ -0,0 +1,170 @@
+package link_test
+
+import (
+	"context"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/almighty/almighty-core/account"
+	"github.com/almighty/almighty-core/gormsupport/cleaner"
+	"github.com/almighty/almighty-core/gormtestsupport"
+	"github.com/almighty/almighty-core/migration"
+	"github.com/almighty/almighty-core/models"
+	"github.com/almighty/almighty-core/resource"
+	testsupport "github.com/almighty/almighty-core/test"
+	"github.com/almighty/almighty-core/workitem"
+	"github.com/almighty/almighty-core/workitem/link"
+
+	"github.com/almighty/almighty-core/space"
+	"github.com/jinzhu/gorm"
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestRunWorkItemLinkRevisionRepositoryBlackBoxTest(t *testing.T) {
+	resource.Require(t, resource.Database)
+	suite.Run(t, &revisionRepositoryBlackBoxTest{DBTestSuite: gormtestsupport.NewDBTestSuite("../../config.yaml")})
```

Comment:
yes, that's something I'd like to see, too. There's already an issue for that: https://github.com/almighty/almighty-core/issues/898, so I'd rather see it done in that issue rather than here.